src/main/java/com/example/CarDealership/models/Dealership.java

```diff
@@ -0,0 +1,57 @@
+package com.example.CarDealership.models;
+
+public class Dealership {
+    private int dealershipId;
+    private String name;
+    private String address;
+    private String phone;
+
+    public Dealership(int dealershipId, String name, String address, String phone) {
+        this.dealershipId = dealershipId;
+        this.name = name;
+        this.address = address;
+        this.phone = phone;
+    }
+
+    public int getDealershipId() {
+        return dealershipId;
+    }
+
+    public void setDealershipId(int dealershipId) {
+        this.dealershipId = dealershipId;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getAddress() {
+        return address;
+    }
+
+    public void setAddress(String address) {
+        this.address = address;
+    }
+
+    public String getPhone() {
+        return phone;
+    }
+
+    public void setPhone(String phone) {
+        this.phone = phone;
+    }
+
+    @Override
+    public String toString() {
+        // Define column widths
```

Comment:
nit: These two variables, idWidth and width, should be final and constants and put up in the same area where there are fields. so private final IDWIDTH = 5; The reason for this is that it's easy for other devs to know these are meant to be constants and if they're with the fields they're easier to adjust.